widget/embedded/src/components/QuoteWarningsAndErrors/SlippageWariningModal.tsx

```diff
@@ -67,6 +68,7 @@ export function SlippageWarningModal(props: PropsTypes) {
         <Divider size={18} />
         <Divider size={32} />
         <Button
+          id="widget-slippage-warning-modal-confirm-anyway-btn"
```

Comment:
In some places, like `UnknownPriceWarningModal`, you use "confirm" for the action name, but in other places, like this one, you've used "confirm-anyway." I think it would be better to keep it consistent.

Reply:
I use the text of the button for easy to detect.